index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Mourid Portfolio</title>
+</head>
+<body>
+    <section></section>
+    <nav class="navbar">
+        <div class="container">
+            <div class="logo">John doe </div>
+        <div class="menu"><img src="img/Union.svg" alt=""></div>
+    </div>
+     </nav>
+    <header class="header">
+        <div class="container">
+                    <div class="textInfo">
+             <h1>Hey There. I'm alex</h1>
+                <h2>I'am a Software Developper</h2>
+                    <p>I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hesitate to contact me.</p>
+                    </div>
+
+        <img src="img/Illustration-header.png" alt="">
```

Comment:
- [ ] I think this section should have social media icons as per the image below, but we are missing them and some other points

<img width="789" alt="1" src="https://user-images.githubusercontent.com/77843050/212501328-0ffcd8ac-5609-4f9a-9f64-5caa7a620975.png">

- [ ] Consider taking a closer look at the original design and make the necessary modification to match the text color, font weight, padding, and every other thing, please
- [ ] You are missing the `LET'S CONNECT` text on your page, please add it too
- [ ] The social icons are not present either, please fix this and resubmit your work